spring-cloud-dataflow-docs/src/main/asciidoc/getting-started-kubernetes.adoc

```diff
@@ -940,29 +1033,42 @@ spec:
   deploymentProperties:
     "spring.cloud.deployer.group": "mystream"
 ----
+====
 
+[[getting-started-kubernetes-create-stream-register-logsink-app]]
+===== Register the `log-sink` Application
 
-10) Register the `log-sink` application version 2.1.0.RELEASE and update your stream to use it
```

Comment:
This step intentionally included a version to highlight the fact the `log-sink` will be updated to the version as part of the `stream update` workflow.

Reply:
OK. I extended one sentence and added another to reinforce that.